README.md

```diff
@@ -63,4 +62,4 @@ To build NIX-MX under macOS please follow the guide provided at [MacOSBuild.md](
 
 Mathworks release a new edition twice a year. We may not be able to keep up with this pace for lack of time or because we do not have access to the current release. 
 
-Often is is only the build configuration that fails. In such cases please check out this [issue](https://github.com/G-Node/nix-mx/issues/172). In case you successsfully built and tested with a newer MATLAB version than was supported please consider submitting the changes via pull-request. Thank you!
\ No newline at end of file
+Often is is only the build configuration that fails. In such cases please check out this [issue](https://github.com/G-Node/nix-mx/issues/172). In case you successsfully built and tested with a newer MATLAB version than was supported please consider submitting the changes via pull-request. Thank you!
```

Comment:
```suggestion
Often it is only the build configuration that fails. In such cases please check out this [issue](https://github.com/G-Node/nix-mx/issues/172). In case you successsfully built and tested with a newer MATLAB version than was supported please consider submitting the changes via pull-request. Thank you!
```